Core/CoreUITests/CoreUITestCase.swift

```diff
@@ -335,6 +335,43 @@ open class CoreUITestCase: XCTestCase {
         let pixel = UIImage(data: Data(base64Encoded: "iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII=")!)!
         mockDataRequest(URLRequest(url: URL(string: "https://instructure-uploads.s3.amazonaws.com/account_70000000000010/attachments/64473710/canvas_logomark_only2x.png")!),
                         data: pixel.pngData()!)
+        mockCalendarRequests()
+    }
+
+    func mockCalendarRequests() {
+        mockData(
+            GetCalendarEventsRequest(
+                contexts: [ContextModel(.course, id: "1")],
+                startDate: Clock.now.addDays(-365),
+                endDate: Clock.now.addDays(365),
+                type: .event,
+                perPage: 99,
+                include: [.submission]
+            ),
+            value: []
+        )
+        mockData(
+            GetCalendarEventsRequest(
+                contexts: [ContextModel(.course, id: "1")],
+                startDate: Clock.now.addDays(-365),
+                endDate: Clock.now.addDays(365),
+                type: .assignment,
+                perPage: 99,
+                include: [.submission]
+            ),
+            value: []
+        )
+        mockData(
+            GetCalendarEventsRequest(
+                contexts: nil,
+                startDate: Clock.now.addDays(-365),
+                endDate: Clock.now.addDays(365),
+                type: .event,
+                perPage: 99,
+                include: [.submission]
+            ),
+            value: []
+        )
```

Comment:
Will these mocks be needed by more than 1 test?

Reply:
Oh I suppose not. I'll remove these.